e2e/__tests__/confirm_API_data_browse.spec.ts

```diff
@@ -1,4 +1,6 @@
 import { test, expect } from '@playwright/test';
+import * as mongoDB from 'mongodb';
```

Comment:
Really picky personal preference but I tend to prefer to name what's being imported, is there a reason you've gone with the wildcard?

Reply:
agreed

Reply:
fixed